app/views/editor/post.html.erb

```diff
@@ -79,6 +79,10 @@
         <h3>Share your work</h3>
       <% end %>
   
+      <% if params[:redirect] == 'question' %>
+        <%= hidden_field_tag 'redirect', 'question' %>
```

Comment:
I see -- so this is temporary? If so, let's leave a clear comment that it should be removed once question show template is complete. Something like `<!-- Notice: Remove this ... -->`. Thanks!